### DISCUSSION TITLE
Why putting up a bounty program?

### DISCUSSION THREAD

**Discussion by hacksider** [hacksider/Deep-Live-Cam#679] [Announcements]
To answer this question, maintaining this repository alone is quite challenging. Therefore, I’m reaching out to the community for support and participation to help the repository grow. By collaborating, we can ensure that the repository remains a valuable resource for everyone. The bounties are open-sourced to benefit the community, encouraging contributions and shared ownership.

![image](https://github.com/user-attachments/assets/f467bc7d-da8a-4f0d-9d5f-eb1634be61d3)

### How are we putting the bounties cost?
Currently, the bounty amounts are set somewhat arbitrarily. However, we aim to increase these amounts over time as I am personally funding them. This approach allows us to start small and scale up as the community grows and more resources become available.

I am always open to suggestions and feedback. Your input is invaluable in shaping the future of this repository. Let’s work together to create something truly beneficial for everyone involved.

### Who can participate in the bounty program?
Anyone! Just commit and let's see if it will reach the main repo!

### How can I get the bounty?
It will be sent via **_Paypal_** so make sure you have a Paypal account

### Are we still allowed to create a pull request outside the bounty program?
I highly encourage it. Not all great ideas comes from us/community but sometimes it grows within you depending on your needs. This programs aim is to gamify the community.

**Comment by AbandonedCart** [comment]
Do you have any plans to open up submitting bounties to others? Definitely not something you would want to do immediately, since it would require some planning on how to ensure the bounty was paid, the request was realistic, etc etc. It is something that may promote involvement on both ends without being the only one funding ideas, though.

**Reply by hacksider** [reply]
I have plans but i'll put it up gradually, you can also put some bounty ideas and if it is good then I'll try to fund it as well. If there's someone willing to fund their own bounty, I can serve as a middleman to ensure it is actually paid.

**Reply by hacksider** [reply]
You can submit your ideas on https://github.com/hacksider/Deep-Live-Cam/discussions/categories/bounty-ideas

![image](https://github.com/user-attachments/assets/5814d927-4a85-41ea-913d-0601bbe91e74)

**Reply by AbandonedCart** [reply]
Another approach would be allowing others to contribute to a bounty (ie. You set a bounty of $20 and I pledge $10, so the bounty goes up to $30). This is better if you want to keep the features specific, but still allows others to help motivate completing it.

**Reply by hacksider** [reply]
Nice idea on that! I'll see what I can find to make those pledges a reality (maybe another gumroad link for it).

**Reply by AbandonedCart** [reply]
Unfortunately, the site that a lot of projects I worked with used (Bountysource) appears to be on hiatus.

**Reply by hacksider** [reply]
I'll just try to make it work with gumroad despite their fees and let's see how it works. By the way, I have put another bounty as well.

**Reply by AbandonedCart** [reply]
Ironically, my Windows PC is a ROG Ally (no webcam or enough ports to add one). I just like the technology and your dedication.

If they hit you with fees, you may want to consider PayPal donations. They offer an option to allow people to donate and cover the fees. You can also have selections for the donation purpose, which would allow setting the various bounty titles to keep it all somewhat organized.

**Reply by hacksider** [reply]
Sad to say, donation button is for non-profit business account only when it comes to paypal.

**Reply by omarsoufiane** [reply]
**PROMOTION**

@hacksider Hi,
I'm the founder of [Bountyhub](https://www.bountyhub.dev), It's a new platform dedicated to solving the exact issues you are facing, as a maintainer of the project you can fund bounties, or you can solve bounties created by other people using your repository (think of it like premium support). Handling bounties can be challenging because many issues might happen like a disagreement between the bounty creator and the pull request author and our core service is to handle these kinds of situations. Also, the bounty hunter needs to be assured that the prize is already available before they start working on an issue. (@AbandonedCart pledging bounties is not currently supported but we plan to add it soon). let me know if you would like to collaborate with us.

**Reply by AbandonedCart** [reply]
Seems there are quite a few options out there, including https://gitpay.me/#/ that was proposed in a few repositories previously using BountySource during its decline. 

@omarsoufiane Nothing against your service, but the conversation was about others being able to contribute to bounties, so it seems like putting the cart before the horse to pitch a 15% fee to handle mediation and then casually mention that you don't even support that yet at the end of it.

**Reply by omarsoufiane** [reply]
**PROMOTION**

@AbandonedCart gitpay is open source and it seems to be no longer maintained, we just launched 1 month ago and we currently have more features than gitpay (contributing to an increase in the bounty price would be available in a couple of weeks). our fees are higher than gitpay because we offer mediation and dispute resolution, which requires us to test pull requests manually. As for the fees, we can offer a much lower percentage for Deep-Live-Cam if @hacksider would like to collaborate with us. we are still new but our users are increasing day after day and we will certainly bring more people to contribute to Deep-Live_Cam.

**Reply by AbandonedCart** [reply]
@omarsoufiane You're pitching to the wrong person, sorry. I was suggesting the concept of allowing users to contribute to bounties.

Also, gitpay was updated 5 days ago. See https://github.com/worknenjoy/gitpay

@KRSHH Thanks for the edit.

**Reply by hacksider** [reply]
Thanks for all the suggestions. I will surely try all of them but since it's more of a personal funding, I will just stick personal payment (I already paid 2 of them 😄 without any issues).
